app.js

```diff
@@ -40,28 +40,32 @@ const jws = require('jws-jwk');
 const request = require('request');
 
 const configServer = config.get('server');
-const fileChoiceUrl = configServer.has('fileChoiceUrl') ? configServer.get('fileChoiceUrl') : '';
+const fileChoiceUrl = configServer.has('fileChoiceUrl') ?
+    configServer.get('fileChoiceUrl') : '';
 const siteUrl = configServer.get('siteUrl');
 const plugins = config.get('plugins');
 const permissionService = require('./permissions');
 
-var token;
-var userEmail;
+let token;
+let userEmail;
 
 process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
 
-String.prototype.hashCode = function () {
-  for (var ret = 0, i = 0, len = this.length; i < len; i++) {
-    ret = (31 * ret + this.charCodeAt(i)) << 0;
+String.prototype.hashCode = function hashCode() {
+  let i;
+  let len;
+  let ret;
+  for (ret = 0, i = 0, len = this.length; i < len; i + 1) {
+    ret = ((31 * ret) + this.charCodeAt(i)) << 0;
   }
   return ret;
 };
-String.prototype.format = function () {
+String.prototype.format = function format() {
   let text = this.toString();
 
   if (!arguments.length) return text;
 
-  for (let i = 0; i < arguments.length; i++) {
+  for (let i = 0; i < arguments.length; i + 1) {
```

Comment:
Our eslint config wants it that way (see http://eslint.org/docs/rules/no-plusplus).
We can of course just change the config, not a big deal.